docs/installation.md

```diff
@@ -0,0 +1,243 @@
+# Installing ansible-navigator with execution environment support
+
+[podman_url]: https://podman.io/
```

Comment:
Traditionally, the detached link definitions live below the content. Either in the bottoms of the paragraphs where they are used or sections or accumulated at the bottom of the whole document. Could you adjust this accordingly?